cmd/nerdctl/logs.go

```diff
@@ -96,6 +99,10 @@ func logsAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	stopChannel := make(chan os.Signal, 1)
+	// catch OS signals:
+	signal.Notify(stopChannel, syscall.SIGTERM, syscall.SIGINT)
```

Comment:
Even if we do manage to silence the linter, adding `SIGKILL` to the catch list would not have any effect since the process would never actually receive the SIGKILL, as it's handled entirely on the OS-side. (i.e. the OS kills the process without it notifying the process in any way)